src/ECS/utilities/Zipper/IndexedZipper.hpp

```diff
@@ -0,0 +1,26 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** IndexedZipper.hpp file
+*/
+
+#ifndef INDEXED_ZIPPER_HPP
+    #define INDEXED_ZIPPER_HPP
+
+    #include "IndexedZipperIterator.hpp"
+    #include "GenericZipper.hpp"
+
+template<class... Containers>
+class IndexedZipper : public GenericZipper<
```

Comment:
Same as for the generic Zipper class, could you add a doxygen comment to this class ?